CalibTracker/SiStripCommon/interface/Book.h

```diff
@@ -74,7 +74,7 @@ class Book {
   void fill( double_t X, double_t Y, const poly<std::string>& names, 
 	     uint_t NbinsX, double_t Xlow, double_t Xup, double_t W=1 )
   { 
-    BOOST_FOREACH(string_t name, std::make_pair(names.begin(),names.end())) {
+    for(auto const& name : names) {
```

Comment:
@guitargeek are you sure that those loops are equivalent? `std::make_pair` will return a pair of first and last elements while the new version will iterate over every value in `names`.

Reply:
Loops are indeed equivalent as discussed above.